<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssoCameraBindPanel</class>
 <widget class="QGroupBox" name="IssoCameraBindPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(255, 255, 255);
	font-size: 9pt;
}

QComboBox
{
	background-color: rgb(230, 230, 230);
	border-radius: 6px;
	border: 1px solid gray;
	min-width: 120px;
	min-height: 18px;
}</string>
  </property>
  <property name="title">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblSmoke">
       <property name="text">
        <string>Дым</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSmoke"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblCO">
       <property name="text">
        <string>Угарный газ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbCO"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblBtnFire">
       <property name="text">
        <string>Кнопка &quot;Пожар&quot;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBtnFire"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lblSecurity">
       <property name="text">
        <string>Кнопка &quot;Охрана&quot;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBtnSecurity"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lblVoltBattery">
       <property name="text">
        <string>АКБ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbVoltBattery"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="lblVoltBus">
       <property name="text">
        <string>Питание</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbVoltBus"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="lblVoltSensors">
       <property name="text">
        <string>Датчики</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbVoltSensors"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="lblTemp">
       <property name="text">
        <string>Температура</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbTemp"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="lblVibration">
       <property name="text">
        <string>Вибрация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbVibration"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lblSensorOpen">
       <property name="text">
        <string>Вскрытие</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSensorOpen"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="lblSupplyInternal">
       <property name="text">
        <string>Сеть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSupplyInternal"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="lblSupplyExternal">
       <property name="text">
        <string>Внешняя сеть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSupplyExternal"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
